### PR TITLE
Activating Open Collective

### PR DESCRIPTION
**Pull request by monkeywithacupcake**
This pull request adds backers and sponsors from your Open Collective https://opencollective.com/rollup ❤️
  
  It adds two badges at the top to show the latest number of backers and sponsors. It also adds placeholders so that the avatar/logo of new backers/sponsors can automatically be shown without having to update your README.md. [[more info](https://github.com/opencollective/opencollective/wiki/Github-banner)]. See how it looks on [this repo](https://github.com/apex/apex#backers).
We have also added a `postinstall` script to let people know after `npm|yarn install` that you are welcoming donations (optional). [[More info](https://github.com/OpenCollective/opencollective-cli)]
You can also add a "Donate" button to your website and automatically show your backers and sponsors there with our widgets. Have a look here: https://opencollective.com/widgets

  P.S: As with any pull request, feel free to comment or suggest changes. The only thing "required" are the placeholders on the README because we believe it's important to acknowledge the people in your community that are contributing (financially or with code!).

  Thank you for your great contribution to the open source community. You are awesome! 🙌
  And welcome to the open collective community! 😊

  Come chat with us in the #opensource channel on https://slack.opencollective.com - great place to ask questions and share best practices with other open source sustainers!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Contribute
+
+## Introduction
+
+First, thank you for considering contributing to rollup! It's people like you that make the open source community such a great community! 😊
+
+We welcome any type of contribution, not only code. You can help with 
+- **QA**: file bug reports, the more details you can give the better (e.g. screenshots with the console open)
```
Comment (lukastaegert) [inline CONTRIBUTING.md]: Maybe
"file bug reports, the more details you can give the better (e.g. [REPL](https://rollupjs.org/repl)-links or repos that demonstrate the specific issue)"
which is more helpful for the kind of issues people usually have with rollup :)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+Working on your first Pull Request? You can learn how from this *free* series, [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github).
+
+## Submitting code
+
+Any code change should be submitted as a pull request. The description should explain what the code does and give steps to execute it. The pull request should also contain tests.
+
+## Code review process
+
+The bigger the pull request, the longer it will take to review and merge. Try to break down large pull requests in smaller chunks that are easier to review and merge.
+It is also always helpful to have some context for your pull request. What was the purpose? Why does it matter to you?
```
Comment (lukastaegert) [inline CONTRIBUTING.md]: Maybe add "Does it resolve any known Github issues?"

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+## Financial contributions
+
+We also welcome financial contributions in full transparency on our [open collective](https://opencollective.com/rollup).
+Anyone can file an expense. If the expense makes sense for the development of the community, it will be "merged" in the ledger of our open collective by the core contributors and the person who filed the expense will be reimbursed.
+
+## Questions
+
+If you have any questions, create an [issue](issue) (protip: do a quick search first to see if someone else didn't ask the same question before!).
+You can also reach us at hello@rollup.opencollective.com.
```
Comment (lukastaegert) [inline CONTRIBUTING.md]: Who will receive these emails? If these only go to @Rich-Harris (who originally applied for Open Collective), this is probably not the best of ideas. My hunch is to remove this section as issues are still the single best way of getting yourself heard.

On the other hand, we are often flooded with support issues that divert our very limited resources from actually fixing or improving things so we might use this opportunity to divert these to other forums:

"If you require technical assistance, [Stackoverflow](https://stackoverflow.com/questions/tagged/rollupjs) or [Gitter](https://gitter.im/rollup/rollup) are usually the best places to start.

You can also create an [issue](issue) (protip: do a quick search first to see if someone else didn't ask the same question before!)."

**Comment by Conduitry** [comment]
I'm really not a fan of these changes. It seems quite presumptuous to make people using Rollup to download a bunch of additional packages and to also use up a bunch of console real estate printing out some giant ASCII art. Installing just Rollup now installs 46 packages instead of just 1. I realize the desire to make people aware of OpenCollective and to donate to Rollup if they are able, but I don't think https://github.com/rollup/rollup/pull/1915/commits/64fad1609aa1f2b97791108035447a7c879fc4a1 is a good way to go about that.

**Comment by lukastaegert** [comment]
I wanted to give it a try to see how it feels but I guess you're right, we should remove this particular change again.

**Comment by lukastaegert** [comment]
I have removed it on master so that it will vanish with the next release.

**Comment by monkeywithacupcake** [comment]
Yes, the postinstall works great for some repos, but it really isn't for everyone. I understand when people don't want to include it. You can definitely ignore that part. If you ever come to the point where you want to reach folks postinstall, there are alternatives that open collective is aware of:

One user made a lighter version of the post install: https://github.com/WebReflection/lightercollective
You can also remove the dependency altogether: https://github.com/remy/nodemon/blob/master/package.json#L42

**Comment by lukastaegert** [comment]
Thanks, I guess the minimal "echo" version might make sense. I decided to release it without the message for now (i.e. v0.55.3 is now live) since I think OpenCollective is now featured very prominently in the Readme as well as the issue template and contribution guidelines.
Might make sense to include it on the website at some point, though.

**Comment by Conduitry** [comment]
Looks like this came back in 0.55.4?

**Comment by lukastaegert** [comment]
I assume one of the PRs somehow brought this back, sorry about that. Just released a small patch that fixes this again.

**Pull request merged**